src/main.rs

```diff
@@ -60,7 +61,11 @@ fn main() {
 
     let matches = match opts.parse(&args[1..]) {
         Ok(m) => m,
-        Err(f) => panic!(f.to_string()),
+        Err(e) => {
+          writeln!(&mut io::stderr(), "{}", e)
+              .expect("failed printing to stderr");
+          std::process::exit(1);
```

Comment:
hm....that doesn't work :( It just exits.

    $ ./target/debug/cobalt -fooo
    $